src/MSA.BuildingBlocks.ServiceClient/README.md

```diff
@@ -11,29 +11,30 @@ To install the package, use the following command in the Package Manager Console
     PM> Install-Package MSA.BuildingBlocks.ServiceClient
 
 ## Usage
-
-First, add the `IServiceClient` to the dependency injection container in `Startup.cs`:
+<details open> <summary>For package version 0.1.0</summary>
```

Comment:
We don't need to keep info for every version - use git history

Reply:
True. Going to change.